new-website/utils/tutorials/export_tutorials.py

```diff
@@ -0,0 +1,182 @@
+"""
+This script is used to parse HTML files containing DeepChem tutorials and convert them into React components
+and JSON data that is used to display tutorials on the website.
+
+Requirements:
+    - BeautifulSoup (bs4)
+    - json
+    - re
+
+Example Usage:
+    - Ensure that the fetch_tutorials.py script has been executed successfully.
+    - Run this script using "python script_name.py".
+    - The output JSON and React components data will be generated in the "../../deepchem/data/tutorials/" and
+      "../../deepchem/pages/tutorials/" directories.
+"""
+from bs4 import BeautifulSoup
+import json
+import re
+
+
+def to_valid_identifier(s):
```

Comment:
Can we add some basic unit tests for this file? It is complicated enough that I think it would be useful to have some tests